polkadotjs-examples/src/from-nathan.ts

```diff
@@ -0,0 +1,28 @@
+import { WebSocket } from "ws";
+
+// using https://github.com/websockets/ws
+async function main() {
+console.log('**** inside main');
+    const ws = new WebSocket("ws://127.0.0.1:9944");
+    ws.on("open", () => {
+        console.log("opened");
+        ws.send('{"id":1,"jsonrpc":"2.0","method":"creditcoin_eventsSubscribe"}');
+    });
+
+console.log('**** after open');
+
+    ws.on("message", (data) => {
+        console.log("received: %s", data);
+    });
+
+console.log('**** after message');
+
+    // just to exit gracefully on ctrl-c
+    process.on("SIGINT", () => {
+        console.log("exitting ...");
+        ws.close();
+        process.exit();
+    });
+}
+
+main().catch(console.error).finally(() => process.exit());
```

Comment:
@nathanwhit this example is still missing something and I'm not sure what it is. The output is:

```bash
$ npx ts-node ./src/from-nathan.ts 
**** inside main
**** after open
**** after message
```

However I don't see the `open` or `received:` console messages anywhere. Will keep digging into it but it's not yet obvious to me what's going on.